src/app_service/service/community/service.nim

```diff
@@ -527,6 +535,10 @@ QtObject:
         let response = tokens_backend.registerLostOwnershipNotification(community.id)
         checkAndEmitACNotificationsFromResponse(self.events, response.result{"activityCenterNotifications"})
 
+      # new community token
```

Comment:
This description doesn't seem to describe exactly what the `communityTokensChanged` does!